### PR TITLE
LEAF601 remediations for admin mod group page

### PR DESCRIPTION
**Pull request by jeanmarcablevets**
clear cache before testing

### CODE REVIEW

File: LEAF_Nexus/templates/permission_iframe.tpl [hunk 1]
```diff
@@ -1,26 +1,26 @@
 <!--{foreach from=$privileges item=privilege}-->
     <!--{if $privilege.read == 0}-->
-<div style="padding: 4px; color: red">
+<div tabindex="0" style="padding: 4px; color: red">
```
Comment (dastan-rahimi) [inline LEAF_Nexus/templates/permission_iframe.tpl]: Please add role="button" so the screen readers read them as buttons

Comment (jeanmarcablevets) [reply LEAF_Nexus/templates/permission_iframe.tpl]: Done

File: LEAF_Nexus/templates/view_group.tpl [hunk 2]
```diff
@@ -13,29 +13,29 @@
...
         </div>
     </div>
 </div>
 
 <div id="toolbar" class="toolbar_right toolbar noprint">
-    <div id="tools"><h1>Options</h1>
+    <div id="tools"><h1 tabindex="0">Options</h1>
         <!--{if array_search('service', $tags) !== false}-->
-        <div onclick="window.location='?a=navigator&amp;rootID=<!--{$groupLeader|sanitize}-->'"><img src="../libs/dynicons/?img=preferences-system-windows.svg&amp;w=32" style="vertical-align: middle" alt="View Org Chart" title="View Org Chart" /> View in Org Chart</div>
+        <div id="view_orgchart" ><a role="button" href="?a=navigator&amp;rootID=<!--{$groupLeader|sanitize}-->"></a><img src="../libs/dynicons/?img=preferences-system-windows.svg&amp;w=32" style="vertical-align: middle" alt="View Org Chart" title="View Org Chart" /> View in Org Chart</div>
```
Comment (jeremyrl) [inline LEAF_Nexus/templates/view_group.tpl]: This doesn't work. Click does nothing.

File: LEAF_Nexus/templates/permission_iframe.tpl [hunk 3]
```diff
@@ -31,9 +31,16 @@
...
     You have access to <u>Edit Permissions</u> for this field.</div>
     <!--{/if}-->
 </div>
-<!--{/foreach}-->
\ No newline at end of file
+<!--{/foreach}-->
+<script>
+    function triggerClick(e, id) {
+        if(e.keyCode === 13) {
+            document.getElementById('editpermissions').click();
```
Comment (nathanws) [inline LEAF_Nexus/templates/permission_iframe.tpl]: It looks like the `id` is being passed in but not used?

**Pull request merged**